### ISSUE TITLE
Добавить уведомление о начале работы над задачей

### ISSUE THREAD

**Issue by capitoshko** [BlizD/Tasks#256]
Предлагаю добавить оповещение для заказчика о начале работы над задачей. При изменении вида статуса с BackLog на ВРаботе можна отправлять уведомление.

**Comment by capitoshko** [comment]
Предлагаю также добавить возможность отправлять уведомления при изменение того, кто может дать ответ.

**Comment by BlizD** [comment]
> Предлагаю добавить оповещение для заказчика о начале работы над задачей. При изменении вида статуса с BackLog на ВРаботе можна отправлять уведомление.

Игорь, согласен. 

> Предлагаю также добавить возможность отправлять уведомления при изменение того, кто может дать ответ.

Согласен.

**Comment by capitoshko** [comment]
Тогда беру в работу

**Comment by milanse** [comment]
Вообще это все хорошо бы параметризировать через РС, чтобы можно было отельный шаблон присвоить на событие.

**Comment by capitoshko** [comment]
Я согласен. Думаю для начала сделать так как делали. Это позволить изучить код и потом  переделать на регистр сведений.

**Comment by BlizD** [comment]
@capitoshko 
По шаблонам сделано следующее https://github.com/BlizD/Tasks/wiki/Как-указать-свой-шаблон-уведомления-для-задач

Игорь, напиши какие изменения метаданных собираешься сделать в рамках этой задачи.

Обращаю внимание, что в справочнике узСтатусыЗадач уже есть реквизит "Заказчик" в ТЧ: Наблюдатели, предлагаю использовать его при настройке уведомления.


![image](https://user-images.githubusercontent.com/10989306/98013728-c7c54580-1e0b-11eb-9beb-1dccd453cfc8.png)

**Comment by capitoshko** [comment]
Спасибо. Посмотрю.
